python/tvm/topi/cuda/scatter.py

```diff
@@ -789,42 +790,80 @@ def gen_ir(data_ptr, indices_ptr, updates_ptr, out_ptr):
 
         # For now we avoid parallizing over dimensions indexed by `indices` as
         # there may be repeated indices and hadling parallel accumulation can
```

Comment:
This comment is not valid anymore right?

Reply:
Fixed